starter-code/pizza.js

```diff
@@ -1 +1,123 @@
 // Write your Pizza Builder JavaScript in this file.
+// $( document ).ready(function() {
+//   $('.btn').click(function(){
+//     $(this).addClass('on');
+//   });
+// });
+
+// // hide/unhide/toggle
+
+// $
+// .add()
+// .append()
+// .remove
+
+// :button
+// .click()
+
+
+// var initialPrice = 10;
+// var totalPrice =
+
+
+$(".btn-pepperonni").click(function(){
+  $(".pep").toggle();
+  $(".price ul li:nth-child(1)").toggle();
+});
+
+$(".btn-mushrooms").click(function(){
+  $(".mushroom").toggle();
+  $(".price ul li:nth-child(2)").toggle();
+});
+
+$(".btn-green-peppers").click(function(){
+  $(".green-pepper").toggle();
+  $(".price ul li:nth-child(3)").toggle();
+});
+
+$(".btn-sauce").click(function(){
+  event.preventDefault();
+  $(".sauce").toggleClass("sauce-white");
+  $(".price ul li:nth-child(4)").toggle();
+});
+
+$(".btn-crust").click(function(){
+  event.preventDefault();
+  $(".crust").toggleClass("crust-gluten-free");
+  $(".price ul li:nth-child(5)").toggle();
+});
+
+$(".btn").click(function() {
+  $(this).toggleClass("active");
+});
```

Comment:
This is actually clever and you're the first one I see using this!
There's this DRY rule in dev (Don't Repeat Yourself), and that makes your code a bit more DRY, by note repeating the toggleClass line in each onclick function, which is good :)